config/routes.rb

```diff
@@ -68,7 +68,6 @@
 
   get '/users/:user/pbs/:game/:category(/*trailing_path)', to: 'users/pbs#show', as: :user_pb
 
-  get   '/games',            to: 'games#index', as: :games
```

Comment:
I know it's temporary until we add the game directory, but can we support this as a redirect to `/search` in the meantime? Just in case bookmarks / tweets / etc. still use it.

Reply:
Yeah, this should also fix the error on the home page then since it's related to `games_path` not existing anymore.